c2corg_ui/templates/index.html

```diff
@@ -37,14 +37,53 @@ <h1 class="welcome-title text-center" translate>The mountain sports community</h
 
     </div>
   </section>
-  <section ng-class="{'show': userCtrl.auth.isAuthenticated()}" class="site-intro">
-    <div class="intro-banner">
-      <h1 class="welcome-title text-center" translate>Activity feed</h1>
-    </div>
-  </section>
 
   <div class="feed">
-    <button class="btn orange-btn" id="add-my-outing" protected-url-btn url="${request.route_path('outings_add')}"><span class="glyphicon glyphicon-plus-sign"></span><span translate>add an outing</span></button>
-    <app-advanced-search document-type="outings" use-map="false"></app-advanced-search>
+    <div class="feed-title flex flexend-align" ng-show="!feedCtrl.busy && !feedCtrl.error">
+      <span class="icon-news"></span>
+      <h3 translate>Activity feed</h3>
+    </div>
+    <h3 translate class="text-center text-danger" ng-show="feedCtrl.error">Sorry, there was an error while getting the feed.</h3>
+
+    <div app-feed app-feed-profile="false" app-loading>
+      <h4 class="text-success text-center" translate ng-show="feedCtrl.busy">Loading feed, please wait...</h4>
```

Comment:
what do you mean? i'm asking myself if i should leave 'app-loading' that will show the spinning wheel as usual for each feed fetch, or just leave the 'loading, please wait...'